app/assets/stylesheets/event_email.css.less

```diff
@@ -0,0 +1,17 @@
+.event_email_show {
+  .well {
+    word-wrap: break-word;
+  }
+
+  .labl {
```

Comment:
Please rename  this one. If it's a label for some specific group of html elements call it *-label:-)